examples/test/test-util.ts

```diff
@@ -33,7 +33,7 @@ export const spaTest = (path: string) => {
 
 export const query = (record: Request) => {
   const query = record.request.url.split('?')[1]
-  return query ? qs.parse(query) : {}
+  return query ? qs.parse(query) : null
```

Comment:
return type of `parse` is `[key: string]: string | string[] | null | undefined;`.
so, it seems that do not good for this function tor return empty object.